src/SellingPartnerApiServiceProvider.php

```diff
@@ -90,8 +86,8 @@ private function registerSingleSeller(): void
             'region' => config('spapi.single.endpoint'),
         ]);
 
-        foreach ($this->provides() as $cls) {
-            $this->app->bind(
+        foreach (SellingPartnerApi::API_CLASSES as $cls) {
+            $this->app->singleton(
```

Comment:
I think that in the merge, this got converted back to `singleton`, unless this was an intentional change